api/pkg/apis/v1alpha1/providers/target/staging/staging.go

```diff
@@ -288,7 +287,7 @@ func (i *StagingTargetProvider) Apply(ctx context.Context, deployment model.Depl
 	jData, _ := json.Marshal(catalog)
 
 	_, err = i.ApiClient.GetCatalogContainer(ctx, containerName, scope, i.Context.SiteInfo.CurrentSite.Username, i.Context.SiteInfo.CurrentSite.Password)
-	if err != nil && strings.Contains(err.Error(), v1alpha2.NotFound.String()) {
```

Comment:
`providers.stage.wait` also includes the same check. Could you please correct it as well?
https://github.com/eclipse-symphony/symphony/blob/379f07758e861b95328d8069a9446d729eface60/api/pkg/apis/v1alpha1/providers/stage/wait/wait.go#L312-L315